src/PowerShellEditorServices/Services/TextDocument/Handlers/TextDocumentHandler.cs

```diff
@@ -40,12 +40,12 @@ class PsesTextDocumentHandler : TextDocumentSyncHandlerBase
             _remoteFileManagerService = remoteFileManagerService;
         }
 
-        public override Task<Unit> Handle(DidChangeTextDocumentParams notification, CancellationToken token)
+        public override Task<Unit> Handle(DidChangeTextDocumentParams request, CancellationToken cancellationToken)
```

Comment:
Worth noting that notifications and requests are separate LSP concepts, this one being a notification. If you already know that and/or still want the change I'm cool with it

Reply:
It _is_ but the base class names it `request` not notification, so there's a Roslynator complaint that the name was changed. We can fix it in OmniSharp (which my god, has a lot of pending analyzer cleanup work that it needs).